lib/senders/partials/new_device_login.html

```diff
@@ -1,6 +1,18 @@
 <% extends "partials/base/base_text_only.html" %>
 
-<% block heading %>{{t "New sign-in to %(clientName)s" }}<% endblock %>
-<% block text_primary %>
+<% block content %>{{t "New sign-in to %(clientName)s" }}<% endblock %>
+<!--Header Area-->
+<tr style="page-break-before: always">
+  <td valign="top">
+    <h1 style="font-family: sans-serif; font-size: 21px; line-height: 29px; font-weight: normal; margin: 0 0 11px 0; text-align: center;">{{t "New sign-in to %(clientName)s" }}</h1>
+    <p class="primary" style="font-family: sans-serif; font-size: 14px; line-height: 21px; font-weight: normal; margin: 0 0 21px 0; text-align: center;">
+<p style="font-family:sans-serif; font-size: 13px; line-height: 20px; font-weight: normal; margin: 0 0 24px 0px; text-align: center; color: #4a4a4f;">
+    {{#if primaryEmail }}{{ primaryEmail }}<br/>{{/if}}
```

Comment:
these additions do not look correct, the template below already has  `<% include "./location/location.html" %>` included. You probably want this here: https://github.com/mozilla/fxa-auth-server/blob/f3261a613780cdec738069d262bc759c046041a3/lib/senders/partials/post_change_primary.html#L12-L32